### ISSUE TITLE
Update latest version in Cocoapods master repo to 0.9.5 to support Xcode 9

### ISSUE THREAD

**Issue by esilverberg** [BrikerMan/BMPlayer#158]
As of 21/9/2017, the latest BMPlayer on Cocoapods.org is 0.9.1, but in order to work out-of-the-box with Xcode 9 and swift 4, we need this change:

https://github.com/BrikerMan/BMPlayer/commit/86524d3af1f8f6879e6a18292b08594eb5332cc3

So that requires 0.9.5

Could you please update Cocoapods with your latest version 0.9.5? Thanks!

**Comment by BrikerMan** [comment]
Please use, will release later~ 
```
pod 'BMPlayer', :git => 'https://github.com/BrikerMan/BMPlayer.git'
```

**Comment by BrikerMan** [comment]
Remember to change NVActivityIndicatorView's swift_version to 3.2 after pod install.

**Comment by BrikerMan** [comment]
Released 1.0.0
